fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIOFork3.scala

```diff
@@ -7,9 +7,10 @@ trait BIOFork3[F[-_, +_, +_]] extends BIOForkInstances {
 }
 
 private[bio] sealed trait BIOForkInstances
+
 object BIOForkInstances extends LowPriorityBIOForkInstances {
   implicit val BIOForkZio: BIOFork3[ZIO] = new BIOFork3[ZIO] {
-    override def fork[R, E, A](f: ZIO[R, E, A]): ZIO[R, Nothing, BIOFiber3[ZIO, E, A]] =
+    @inline override def fork[R, E, A](f: ZIO[R, E, A]): ZIO[R, Nothing, BIOFiber3[ZIO, E, A]] =
```

Comment:
Please do not use inline here as there is little chance for this method to be inlined (there must be a direct call using the val `BIOForkZIO.fork` - which is unlikely as most calls are polymorphic against some parameter `bioFork: BIOFork[F]` - in addition I'm not sure if anonymous classes in `val`s can participate in inlining, possibly you would need to make this an `object` for inline to trigger)

Reply:
Understood, thanks.